apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -168,8 +169,21 @@ export class DetailViewContents extends React.Component {
   };
 
   handleStatusChange = (event) => {
+    const workshopNotAssigned = !this.props.applicationData.pd_workshop_id && !this.props.applicationData.fit_workshop_id;
```

Comment:
Generally speaking I really dislike variables with negative meaning. Its like naming a variable `isFalse` - forces you to do a lot of mental gymnastics when reading the code. Suggest inverting the meaning of this and everything that follows from it

Reply:
I also wonder if assigned_workshop should be an attribute in the teacher application serializer that returns either the FIT workshop or regular workshop depending on application type